src/components/LocationPage/LocationPageHeading.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { Fragment } from 'react';
+import { Region, State, County } from 'common/regions';
+
+const LocationPageHeading: React.FC<{ region: Region }> = ({ region }) => {
+  if (region instanceof State) {
+    return <strong>{region.name}</strong>;
+  }
+
+  if (region instanceof County) {
+    return (
+      <Fragment>
+        <strong>{region.name}, </strong>
+        {region.state.stateCode}
+      </Fragment>
+    );
+  }
```

Comment:
Note: The old code had a link on the state code, I think that we don't need that, but I can add it back if we think that that's useful.

Reply:
I definitely use it sometimes.  So I'd keep it unless there's a specific reason you're removing it.

At some point it'd be nice to have breadcrumbs. 😁 

USA > Washington > King County

Reply:
adding it back!